roles/tower/templates/launch_awx.yaml.j2

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ meta.name }}-launch-awx
+  namespace: {{ meta.namespace }}
+  labels:
+    app: tower
+data:
+  launch-awx-task: |
+    #!/usr/bin/env bash
+    if [ `id -u` -ge 500 ]; then
+        echo "awx:x:`id -u`:`id -g`:,,,:/var/lib/awx:/bin/bash" >> /tmp/passwd
+        cat /tmp/passwd > /etc/passwd
+        rm /tmp/passwd
+    fi
+
+    source /etc/tower/conf.d/environment.sh
+
+    ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m wait_for -a "host=$DATABASE_HOST port=$DATABASE_PORT" all
+    ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m wait_for -a "path=/var/run/redis/redis.sock" all
+
+
```

Comment:
I'm not sure if this is a particular style decision, but I typically keep one newline spacing in shell scripts. Unless there's a good reason not to, can you remove the extra newline here?